ivy_tests/test_ivy/test_frontends/test_numpy/test_ndarray/test_ndarray.py

```diff
@@ -2881,6 +2881,52 @@ def test_numpy_instance_array_wrap__(
     )
 
 
+@handle_frontend_method(
+    class_tree=CLASS_TREE,
+    init_tree="numpy.array",
+    method_name="take",
+    dtype_x_indices_axis=helpers.array_indices_axis(
+        array_dtypes=helpers.get_dtypes("numeric"),
+        indices_dtypes=["int32", "int64"],
+        min_num_dims=1,
+        max_num_dims=5,
+        min_dim_size=1,
+        max_dim_size=10,
+        indices_same_dims=True,
+        valid_bounds=False,
+    ),
+    mode=st.sampled_from(['clip', 'wrap', 'raise']),
+    test_with_out=st.just(False),
+)
```

Comment:
To speed up the process, I won't review this part thoroughly in this review pass. Please look into the checklist you will find in the comments of the PR and try your best to apply all bits of the checklist here.

Don't worry if you can't understand everything, once you think you resolved all of the checklist (excluding the bits you don't understand) please request a review from me